pkg/networkserver/downlink.go

```diff
@@ -751,8 +760,19 @@ func (ns *NetworkServer) processDownlinkTask(ctx context.Context) error {
 						maxUpLength,
 					)
 					if err != nil {
+						if errors.Resemble(err, errScheduleTooSoon) {
+							nextDownlinkAt = dev.MACState.LastConfirmedDownlinkAt.Add(deviceClassCTimeout(dev, ns.defaultMACSettings))
```

Comment:
What is the role of confirmed downlink here?

Reply:
![2019-03-14-08:46:53-screenshot](https://user-images.githubusercontent.com/12877905/54339651-c2632800-4635-11e9-8d48-72b93705355b.png)
LoRaWAN 1.1 spec Chapter 17